core/Visualization/Sparkline.php

```diff
@@ -70,8 +70,26 @@ public function main() {
             }
             $values[] = $value;
         }
-        $sparkline->setData($values);
 
+        $hasFloat = false;
+        foreach ($values as $value) {
+            if (is_float($value)) {
```

Comment:
is there a chance that we are dealing with strings that are floats like `'5.51'`?

Reply:
I suppose it's possible, will update.

Reply:
Updated.

Reply:
I would maybe also add an `is_numeric($value)` maybe to avoid `Warning: A non-numeric value encountered in /in/00aDd on line 3`? https://3v4l.org/00aDd

Not sure what can happen... like

`if ($value && is_numeric($value){  $value = $value + 0; if () {is_float($value){...}  }`

Otherwise it seems to work re strings and float comparison

Reply:
👍

Reply:
Updated.